addons/account/tests/test_account_invoice_report.py

```diff
@@ -115,7 +115,7 @@ def test_invoice_report_multiple_types(self):
             # amount_total  price_average   price_subtotal  residual    quantity
             [2000,          2000,           2000,           2000,       1],
             [1000,          1000,           1000,           1000,       1],
-            [1000,          1000,           1000,           1000,       3],
+            [1000,          333.3333333333333,           1000,           1000,       3],
```

Comment:
Dude... plz fix the indent of the whole assert or something else but don't fuck up the alignment like this ... :cry:

Reply:
indeed it was a hit in the eye :+1: